src/xdocs/checks.xml

```diff
@@ -165,7 +165,7 @@
       </tr>
       <tr>
         <td><a href="config_blocks.html#EmptyBlock">EmptyBlock</a></td>
-        <td>Checks for empty blocks.</td>
+        <td>Checks for empty blocks. This check does not validate sequential blocks.</td>
```

Comment:
All descriptions in this file should just be first sentence.
Please revert this change.